openquakeplatform/bin/deploy.sh

```diff
@@ -591,27 +590,24 @@ oq_platform_install () {
     chmod a+x "/usr/sbin/openquakeplatform"
     mkdir -p /etc/openquake/platform/media
 
+    service apache2 restart
+
     if [ "$GEM_IS_INSTALL" == "y" ]; then
         db_user_create "$gem_db_user" "$gem_db_pass"
         db_base_create "$gem_db_name" "$gem_db_user"
         db_gis_create  "$gem_db_name"
-    fi
 
-    #
-    #  database population (fixtures)
-    for app in "${GEM_APP_LIST[@]}"; do
-        if function_exists "${app}_fixtureupdate"; then
-            "${app}_fixtureupdate" "$oqpdir"
-        fi
-    done
-
-    if [ "$GEM_IS_INSTALL" != "y" ]; then
-        echo "WARNING: this operation could be destructive for the current version of oqplatform database schema, do proper backup before proceeding."
-        read -p "Open a new terminal, migrate the application database schema manually and then press <enter> to continue: " qvest
-    fi
-    if [ "$GEM_IS_INSTALL" == "y" ]; then
+        #
+        #  database population (fixtures)
+        for app in "${GEM_APP_LIST[@]}"; do
+            if function_exists "${app}_fixtureupdate"; then
+                "${app}_fixtureupdate" "$oqpdir"
+            fi
+        done
         openquakeplatform syncdb --all --noinput
     else
+        echo "WARNING: this operation could be destructive for the current version of oqplatform database schema, do proper backup before proceeding."
+        read -p "Open a new terminal, migrate the application database schema manually and then press <enter> to continue: " qvest
         openquakeplatform syncdb --noinput
     fi
 
```

Comment:
Add a note about the **installation** and **updates** of **old** application and **new** application.